### DISCUSSION TITLE
Documentation missing include in wp-config.php ?

### DISCUSSION THREAD

**Discussion by icepicknz** [stuttter/ludicrousdb#178] [Q&A]
Hi & thanks for the contribution to a great plugin.

I just wanted to check if the documentation wiki was missing a step of including 
/** Load LudicrousDB class */
require_once( dirname( __FILE__ ) . '/wp-content/db.php' );

I have placed this below the require_once ABSPATH . 'wp-settings.php';

I have not activated the plugin in wp plugins web interface, but had tried and I wasn't seeing requests to the other database nodes; I am since adding this require_once.

Please advise if I have done something wrong or if this is indeed a missed step, or I missed reading it :)

**Comment by JJJ** [comment]
Hi there 👋

Manually including that file should not be necessary.

It is a special kind of "drop-in plugin" and is included automatically by WordPress, in the `require_wp_db()` function, via `wp-settings.php`.

LudicrousDB will not proceed if the `$wpdb` global is set (or is not `null`), so check around and see if another plugin (or some other code) is manually setting that up ahead of time.
